release/staging-build.yml

```diff
@@ -186,6 +186,13 @@ batch:
         variables:
           PROJECT_PATH: projects/brancz/kube-rbac-proxy
           CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/brancz.kube-rbac-proxy
+    - identifier: canonical_ubuntu
+      env:
+        type: ARM_CONTAINER
+        compute-type: BUILD_GENERAL1_SMALL
+        variables:
+          PROJECT_PATH: projects/canonical/ubuntu
+          CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/canonical.ubuntu
```

Comment:
What's in this repository?

Reply:
Removed it by setting REPO_NO_CLONE=true in Makefile.